plugins/inputs/systemd_units/README.md

```diff
@@ -1,45 +1,9 @@
-# systemd Units Input Plugin
+# Systemd-Units Input Plugin
 
-The systemd_units plugin gathers systemd unit status on Linux. It uses the
-`systemctl list-units` or the `systemctl show` command to collect data on
-service status.
+This plugin gathers the status of systemd-units on Linux, using systemd's DBus
+interface.
 
-This plugin is related to the [win_services module](../win_services/README.md),
-which fulfills the same purpose on windows.
-
-This plugin supports two modes of operation:
-
-## Using `systemctl list-units`
-
-This is the default mode. It uses the output of
-`systemctl list-units [PATTERN] --all --plain --type=service` to collect data on
-service status.
-
-This mode will not supply as much information as `systemctl show`, but will be
-compatible with almost every version of systemd.
-
-The results are tagged with the unit name and provide enumerated fields for
-loaded, active and running fields, indicating the unit's health.
-
-In addition to services, this plugin can gather other unit types as well,
-see `systemctl list-units --all --type help` for possible options.
-
-## Using `systemctl show`
-
-This mode can be enabled by setting the configuration option `subcommand` to
-`show`. The plugin will use
-`systemctl show [PATTERN] --all --type=service --property=...` to collect data
-on service status.
-
-This mode will yield more data on the service status. See the metrics chapter
-for a list of properties.
-
-The results are tagged with the unit name, unit status, preset status and
-provide enumerated fields for loaded, active and running fields, as well as the
-restart count and memory usage of the unit.
-
-In addition to services, this plugin can gather other unit types as well,
-see `systemctl show --all --type help` for possible options.
+Please note: At least systemd v230 is required!
```

Comment:
Is this a new requirement due to this PR? I think the only major distro version this affects is Ubuntu Xenial, which is not in standard support, but wanted to confirm.

Reply:
The requirements comes from the DBUS API which can only list specific units starting from this version. I'm pretty sure this is also true for the systemctl command shown as it also uses this API AFAICS. I could try to work-around that requirement, but this means we will not get the state for disabled units with older versions.

Reply:
Nah, then this is good to go if it may already be required.